test-app/tests/integration/components/combobox-test.js

```diff
@@ -215,7 +215,7 @@ module('Integration | Component | <Combobox>', function (hooks) {
         await render(hbs`
           <Combobox 
             @value={{this.value}} 
-            @onChange={{this.setValue}} 
+            @onChange={{this.onChange}} 
```

Comment:
In relevant test we specified `onChange` callback, not `setValue` (click expand up button)